src/containers/backpack.jsx

```diff
@@ -132,8 +136,8 @@ const getTokenAndUsername = state => {
     // Look for the session state provided by scratch-www
     if (state.session && state.session.session) {
         return {
-            token: state.session.session.token,
-            username: state.session.session.username
+            token: state.session.session.user.token,
+            username: state.session.session.user.username
```

Comment:
While configuring project, asset, and backpack hosts together I discovered this path was incorrect. I can pull this out to a separate PR if you want.